builder/proxmox/common/config.go

```diff
@@ -171,6 +172,59 @@ type vgaConfig struct {
 	Memory int    `mapstructure:"memory"`
 }
 
+// Allows passing through a host PCI device into the VM. For example, a graphics card
+// or a network adapter. Devices that are mapped into a guest VM are no longer available
+// on the host. A minimal configuration only requires either the `host` or the `mapping`
+// key to be specifed.
+//
+// Note: VMs with passed-through devices cannot be migrated.
+//
+// Example configuration (HCL):
+//
+// ```hcl
+// pci_devices {
+// host          = "0000:0d:00.1"
+// pcie          = false
+// device_id     = "1003"
+// legacy_igd    = false
+// mapping       = "someNic"
+// mdev          = "some-model"
+// hide_rombar   = false
+// romfile       = "vbios.bin"
+// sub_device_id = ""
+// sub_vendor_id = ""
+// vendor_id     = "15B3"
+// x_vga         = false
+// }
+//
+//	```
```

Comment:
Because of the tab here I would think this won't render as expected, I suggest this is placed on the same level as the beginning of the code block.